src/main/java/org/fxmisc/flowless/VirtualFlowContent.java

```diff
@@ -169,6 +170,14 @@ void scrollBreadth(double deltaBreadth) {
         setBreadthOffset(breadthOffset.get() + deltaBreadth);
     }
 
+    void scrollLengthToPixel(double pixel) {
+        setLengthOffset(pixel);
+    }
+
+    void scrollBreadthToPixel(double pixel) {
+        setBreadthOffset(pixel);
+    }
```

Comment:
Written them? Yes. Knew that by writing them that way that that was what I was doing? No.
I had only seen the three modifiers `public`, `protected`, and `private`, so I only thought there were three, lol... I looked it up and it all makes sense now.